### PR TITLE
I have updated the css part in that i have added css units

### PR DESCRIPTION
**Pull request by Rishabh32700**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [Style Stage - A modern CSS showcase styled by community contributions](https://stylestage.dev/)
 - :video_camera: [Tailwind CSS Tutorial for Beginners - Full Course](https://www.youtube.com/watch?v=4wGmylafgM4)
 - :video_camera: [Introduction to CSS](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH9D0J0Mtp6lIiD_8046k3si)
+- :video_camera: [CSS Units in Emglish](https://nados.io/video/css-units-px-rem-em-vh-vw-absolute-vs-relative-units)
```
Comment (shahednasser) [inline README.md:226]: Please change the title of the link to the title of the page

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @Rishabh32700 for your contribution, I just have one comment below.

File: README.md (hunk 1 above)
Comment (Rishabh32700) [inline README.md:226]: :video_camera: [CSS Units px, rem, em, vh, vw | Absolute Vs Relative Units] (https://nados.io/video/css-units-px-rem-em-vh-vw-absolute-vs-relative-units)

Comment (Rishabh32700) [reply README.md:226]: hey plz check now if it is fine or not?

Comment (shahednasser) [reply README.md:226]: Yes that's how it should be

Comment (Rishabh32700) [reply README.md:226]: so, should I generate another PR to get it merged?
actually I'm new to open source can you plz help me out i just wanna know should i generate new PR or have to edit somewhere here?

**Comment by shahednasser** [comment]
Thank you @Rishabh32700 all you need to do when making changes is commit the changes to your forked repository and it will appear here.

**Pull request merged**